libraries/AC_Avoidance/AP_OAPathPlanner.h

```diff
@@ -54,6 +54,14 @@ class AP_OAPathPlanner {
         OA_PATHPLAN_DIJKSTRA = 2
     };
 
+    // enumeration for _OPTION parameter
+    enum OARecoveryOptions {
+        OA_OPTION_DISABLED = 0,
+        OA_OPTION_WP_RESET = (1 << 0),
+    };
+
+    uint16_t get_options_on_recovery() const { return _options;}
```

Comment:
might be worth changing to just get_options(), no big deal tho, you can always change it when you add some more